openstack/utils/templates/_linkerd.tpl

```diff
@@ -10,3 +10,17 @@ ingress.kubernetes.io/service-upstream: "true"
 nginx.ingress.kubernetes.io/service-upstream: "true"
 {{- end }}
 {{- end }}
+
+# Place this in job scripts when your script stops normally, but not abnormally
+# as this causes the side-car pod finish normally, but we need it for the re-runs
+{{- define "utils.linkerd.signal_stop_script" }}
+{{- if and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested }}
+if which curl > /dev/null; then
```

Comment:
Maybe better `command -v curl > /dev/null`. Here is the [reasoning](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script)